lib/active_record_shards/connection_switcher-5-0.rb

```diff
@@ -20,9 +20,15 @@ def ensure_shard_connection
 
       pool = connection_handler.retrieve_connection_pool(spec_name)
       if pool.nil?
-        resolver = ActiveRecord::ConnectionAdapters::ConnectionSpecification::Resolver.new configurations
-        spec = resolver.spec(spec_name.to_sym, spec_name)
-        connection_handler.establish_connection spec
+        spec = if ActiveRecord::VERSION::MINOR.zero?
```

Comment:
also this is kind of wrong ... would also work on 4.0 etc ... so do if MAJOR == 5 && MINOR == 0 ?